odbc/src/api/functions.rs

```diff
@@ -328,7 +361,7 @@ pub unsafe extern "C" fn SQLBrowseConnectW(
     _buffer_length: SmallInt,
     _out_buffer_length: *mut SmallInt,
 ) -> SqlReturn {
-    unimpl!(connection_handle);
+    unsupported_function!(MongoHandleRef::from(connection_handle))
```

Comment:
oh, thank you for reminding why we have the distinction.
I couldn't figure it out and because unsupported_function is in data.rs, I thought we just mixed things up.
We indeed need 2 macros!